app/src/main/java/com/infomaniak/swisstransfer/ui/utils/ContextExt.kt

```diff
@@ -58,3 +58,30 @@ fun Context.openAppNotificationSettings() {
         }
     }.also(::startActivity)
 }
+
+fun Context.copyText(text: String, showSnackbar: (String) -> Unit) {
+    val clipboardManager = getSystemService(Context.CLIPBOARD_SERVICE) as ClipboardManager
+    clipboardManager.setPrimaryClip(ClipData.newPlainText(text, text))
+
+    showSnackbar(getString(R.string.snackbarLinkCopiedToClipboard))
```

Comment:
We only need to provide feedback for android version < 13, because the sytstem takes care of it starting from Android 13

Reply:
I don't know to what extent this applies to all manufacturer but the document does indeed seem to say that it's a standard so yeah I agree maybe we should skip it for API 13+: https://developer.android.com/develop/ui/views/touch-and-input/copy-paste#Feedback